<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskCreateDialog</class>
 <widget class="QDialog" name="TaskCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelSkill">
     <property name="text">
      <string>Skill that task belong to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxSkillChoice"/>
   </item>
   <item>
    <widget class="QLabel" name="labelParentTask">
     <property name="text">
      <string>Parent Task</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxParentTaskChoice"/>
   </item>
   <item>
    <widget class="QLabel" name="labelTaskName">
     <property name="text">
      <string>Brief description of the task</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditTaskName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelTaskDesc">
     <property name="text">
      <string>Full description of the task</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditTaskDesc"/>
   </item>
   <item>
    <widget class="QLabel" name="labelTaskScore">
     <property name="text">
      <string>Score for task</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditTaskScore">
     <property name="text">
      <string>50</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskCreateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskCreateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
